sub/sd_ass.c

```diff
@@ -450,8 +450,16 @@ static void configure_ass(struct sd *sd, struct mp_osd_res *dim,
     ass_set_hinting(priv, set_hinting);
     ass_set_line_spacing(priv, set_line_spacing);
 #if LIBASS_VERSION >= 0x01600010
-    if (converted)
+    if (converted) {
         ass_track_set_feature(track, ASS_FEATURE_WRAP_UNICODE, 1);
+        if (!opts->sub_vsfilter_bidi_compat) {
```

Comment:
So this is not enabled upstream and instead we enable it in mpv behind an compat option that can be enabled?

Reply:
Correct, however we only do it for converted subs because ASS files might break if we enable it for them.

Reply:
So there is no way to do it for ASS subttiles. Wouldn't it be better to enable sub-vsfilter-bidi-compat=yes and treat ass uniformly? converted check shouldn't be extended to more and more behavior differences

Reply:
It can be, but it will almost definitely break ASS subtitles.

Reply:
ok, leave it as-is now. we can monitor what are the needs and what breaks.